### ISSUE TITLE
Asset-Builder does not accept rgba png files

### ISSUE THREAD

**Issue by nerathul** [pimoroni/32blit-beta#13]
tried to get a rgba png file formated from aseprite to use in code, get error from line 207 -> too many arguments, expected 3.
![failureAssetBuilder](https://user-images.githubusercontent.com/35363344/69992940-12c74080-154b-11ea-890b-4295e6305ad2.PNG)
![wall](https://user-images.githubusercontent.com/35363344/69992958-1d81d580-154b-11ea-80de-049f22385d71.png)

png is just 16x16px.

**Comment by Gadgetoid** [comment]
I've updated `asset-builder` to attempt to paletize RGB and RGBA images. Since RGB and RGBA output formats are currently not supported, this means that any RGB or RGBA image that uses 255 or fewer colours will be crunched to an indexed palette format which is currently the one we use in most (all?) examples. This represents a significant size decrease, since RGBA and RGBA are 32bpp and 24bpp respectively whereas paletted is at most 8bpp.

Note: the `--format` parameter currently does nothing, and was indended to control the output format between unpacked RGB/RGBA and paletted.

**Comment by LordEidi** [comment]
The problem I noticed using the asset-builder is that the PNG (with alpha) will get a format of RGB565 (2) in the generated C file. If you try to load that, the engine will crash with a SIGSEGV in surface.cpp on line 32. I noticed that the bounds the line before are wrong (0,0).

When I change the format to 3 (P) by hand and then run the app, everything runs fine (although the coordinates are strange, but that is a different problem).
